### DISCUSSION TITLE
I can't log in after update, what should I do now?

### DISCUSSION THREAD

**Discussion by JetGum** [datenstrom/community#475] [Ask a question] [answered]
Hi,

I login into my Yellow backend on my live server. The update dialogue tells me that there is a long list of files that need to be updated (even the Theme files I don't use).

When I click on "Update" I get **kicked out of the backend**.

I see the "Login failed. E-Mail or Password is wrong. Try again" - Message. 

Plus **I cannot login anymore**.

What's going on?

**Comment by markseuffert** [comment, marked as answer]
I would reset the password or [create a new user account](https://github.com/annaesvensson/yellow-edit#how-to-create-a-user-account). Additionally, I would check if there are any errors in file `system/extensions/yellow-website.log`. I am only guessing what's going on, it sounds like your user accounts have not been converted to the new file format.

**Comment by JetGum** [comment]
PW reset of admin did the trick. No errors in logfile though.

Seems I had a little confusion with the user/admin account, probably because I installed Yellow on localhost (and created an admin) and then transferred the files onto the live server. Could be that the confirmation mail for admin/user has been missing or Apache on localhost has not been configured to send out emails for confirmation.

Maybe the best way to create admin and user accounts would be online.

Thx for the support.
